tests/voq/test_voq_ipfwd.py

```diff
@@ -0,0 +1,1254 @@
+            check_packet(sonic_ping, ports, 'portC', 'portA', dst_ip_fld='nbr_lb', size=size, ttl=ttl, ttl_change=1)
+            check_packet(sonic_ping, ports, 'portD', 'portA', dst_ip_fld='nbr_lb', size=size, ttl=ttl, ttl_change=1)
+
+            vm_host_to_A = nbrhosts[ports['portA']['nbr_vm']]['host']
+
+            check_packet(sonic_ping, ports, 'portA', 'portD', dst_ip_fld='nbr_lb', dev=ports['portD']['asic'],
+                         size=size, ttl=ttl)
+            check_packet(eos_ping, ports, 'portD', 'portA', dst_ip_fld='my_ip', src_ip_fld='nbr_lb',
+                         dev=vm_host_to_A, size=size, ttl=ttl, ttl_change=1)
+
+            # inband
+            check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='inband', size=size, ttl=ttl, ttl_change=0)
+            check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='inband', dst_ip_fld='nbr_ip', size=size,
+                         ttl=ttl, ttl_change=0)
+            check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='inband', dst_ip_fld='nbr_lb', size=size,
+                         ttl=ttl, ttl_change=0)
+
+            # DUT loopback
+            # these don't decrement ttl
+            check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='my_lb_ip', dst_ip_fld='my_ip', size=size,
+                         ttl=ttl, ttl_change=0)
+            check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='my_lb_ip', dst_ip_fld='nbr_ip', size=size,
+                         ttl=ttl, ttl_change=0)
+            check_packet(sonic_ping, ports, 'portA', 'portA', src_ip_fld='my_lb_ip', dst_ip_fld='nbr_lb', size=size,
+                         ttl=ttl, ttl_change=0)
+
+            vm_host_to_A = nbrhosts[ports['portA']['nbr_vm']]['host']
+            check_packet(eos_ping, ports, 'portA', 'portA', dst_ip_fld='my_lb_ip', src_ip_fld='nbr_lb',
+                         dev=vm_host_to_A, size=size, ttl=ttl, ttl_change=0)
+
+            # end to end
+            vm_host_to_A = nbrhosts[ports['portA']['nbr_vm']]['host']
+            check_packet(eos_ping, ports, 'portB', 'portA', dst_ip_fld='my_lb_ip', src_ip_fld='nbr_lb',
+                         dev=vm_host_to_A, size=size, ttl=ttl, ttl_change=0)
+            check_packet(eos_ping, ports, 'portC', 'portA', dst_ip_fld='my_lb_ip', src_ip_fld='nbr_lb',
+                         dev=vm_host_to_A, size=size, ttl=ttl)
+            check_packet(eos_ping, ports, 'portD', 'portA', dst_ip_fld='my_lb_ip', src_ip_fld='nbr_lb',
+                         dev=vm_host_to_A, size=size, ttl=ttl)
+
+
+@pytest.mark.parametrize('port, ip', [('portA', 'my_ip'), ('portA', 'my_lb_ip'), ('portA', 'inband'),
+                                      ('portD', 'my_ip'), ('portD', 'my_lb_ip'), ('portD', 'inband')])
+@pytest.mark.parametrize('version', [4, 6])
+@pytest.mark.parametrize('porttype', ["ethernet", "portchannel"])
+def test_ipforwarding_jumbo_to_dut(duthosts, all_cfg_facts, tbinfo, ptfhost, porttype, version, port, ip,
```

Comment:
I am wondering how much time is required to run this script? Do you have such data?
Take this case as an example, it will be parameterized to 6x2x2=24 cases. There are many such cases in this script.

Reply:
Hi, this test takes 17 to 18 minutes to run 130 cases, not counting precheck.

Reply:
@wangxin, do you see any other scope for optimizing the test execution time here? If this is acceptable, can we get it merged?